docs/griptape-tools/official-tools/google-gmail.md

```diff
@@ -0,0 +1,48 @@
+# GoogleGmailClient
+
+The `GoogleGmailClient` tool provides a way to interact with the GMail API. It can be used to create draft emails, send emails, and more.
+
+```python
+from griptape.tools import GoogleGmailClient
+from griptape.structures import Agent
+
+# Create the GoogleGmailClient tool
+gmail_tool = GoogleGmailClient(
+    service_account_credentials={
+        "type": "service_account",
+        "project_id": "YOUR_PROJECT_ID",
```

Comment:
Example must run in integration tests. Replace dummy values like `"YOUR_PROJECT_ID"` with environment variables via `os.getenv("GOOGLE_PROJECT_ID")`